app/Views/Templates/components/actions/dropdown.blade.php

```diff
@@ -0,0 +1,36 @@
+@props([
+    'buttonClass' => 'btn btn-primary',  // Default button class
```

Comment:
Please make sure to use attributes from our attribute list. The buttonClass consists of multiple attributes: 

variant [regular, header, card]
content-role [primary, secondary, tertiary]
state [info, danger, warning etc]

In this case the dropdowns would also have position to align them with the button

Reply:
By allowing devs to rewrite the class entirely you force them to write css. So we would need to keep the head dropdown css when in actuality it is just a modifier that we should add. In addition to that you wouldn't add any of the daisyui classes and so we'd have none of the standard dropdown functionality.